// A protocol definition to allow to sending chirp values back and forth.

syntax = "proto2";

package minig;

message DDS {
    required uint32 mult = 1;
    required uint32 clk_transition_hex = 2;
    required uint32 chirp_start_hex = 3;
    required uint32 chirp_stop_hex = 4;
    required uint32 mw_time_hex = 5;
    required uint32 mw_freq_hex = 6;
    required uint32 chirp_time_hex = 7;
    required uint32 chirp_freq_hex = 8;
    required float actual_chirp = 9;
    required float detuning = 10;
}

message FringeParams {
    repeated DDS k_up_params = 1;
    repeated DDS k_down_params = 2;
}

